auto_route/example/lib/mobile/screens/home_page.dart

```diff
@@ -55,6 +55,7 @@ class HomePageState extends State<HomePage> {
     // updates
     // we need it to indicate which NavigationRailDestination is active
     return kIsWeb && false
+        // ignore: dead_code
```

Comment:
Is there a lot of dead code in this library? If so, shall we remove it at all?

Reply:
This is a question for @Milad-Akarie.